### PR TITLE
SC-777 Don't fail if hook installation directory is not writable

### PR DESCRIPTION
**Pull request by marek-obuchowicz**

### CODE REVIEW

File: src/GitHook/Composer/Scripts/HookInstaller.php [hunk 1]
```diff
@@ -57,6 +57,7 @@ public static function installProjectHooks(Event $event)
         $hookDirectory = $vendorDir . '/spryker/git-hook/hooks/project/';
         $gitHookDirectory = $vendorDir . '/../.git/hooks/';
 
+        if (!self::checkDirectoryPermissions($gitHookDirectory, $event)) return true;
```
Comment (stereomon) [inline src/GitHook/Composer/Scripts/HookInstaller.php]:
```
if (!self::checkDirectoryPermissions($gitHookDirectory, $event)) {
     return true;
}
```

For the other cases as well please.

**Review by stereomon — CHANGES_REQUESTED** [review]

**Comment by dereuromark** [comment]
Fairly certain the fixer could fix this itself even ( https://travis-ci.org/spryker/git-hook/jobs/545184428 )

**Comment by marek-obuchowicz** [comment]
@dereuromark :+1: thanks for onboarding into validations topic, that's still new for me

**Review by stereomon — APPROVED** [review]
Please add proper release template for this 

https://release.spryker.com/templates/index-core?module=GitHook

**Comment by dereuromark** [comment]
Since we dont release this (merge only), I think only the mergeonly label is missing IMO.

**Review by alex-galych — APPROVED** [review]

**Pull request merged**